packages/components/src/internal/actions.ts

```diff
@@ -565,12 +565,27 @@ export function exportRows(type: EXPORT_TYPES, exportParams: Record<string, any>
     // POST a form
     const form = $(`<form method="POST" action="${url}">`);
     $.each(params, function (k, v) {
-        form.append($(`<input type="hidden" name="${k.toString()}" value="${v}">`));
+        form.append($(`<input type="hidden" name="${k.toString()}" value="${quoteEncodedValue(v)}">`));
     });
     $('body').append(form);
     form.trigger('submit');
 }
 
+const QUOTE_REGEX = new RegExp('"', 'g');
+const QUOTE_ENTITY = '&quot;';
+
+// Issue 45366: form value containing unescaped quotes gets truncated
+export function quoteEncodedValue(rawValue: any) {
+    let safeValue = rawValue;
+
+    const flattenedValue = rawValue instanceof Array ? rawValue[0] : rawValue;
```

Comment:
I assume it won't matter that the safeValue we return is not an array if the rawValue was an array?